setup.py

```diff
@@ -335,7 +335,7 @@ def get_config_var(name):
 # ==========
 setup(
     name='Cartopy',
-    version='0.14.4.dev0',
+    version='0.15.0',
```

Comment:
@lbdreyer Totally not your doing btw, but it seems a little crazy to me that devs need to remember to do this on release in several places ...

The `cartopy.__version__` should suffice here. It's completely discoverable.

If you were to implement a scheme similar to iris, see [here](https://github.com/SciTools/iris/blob/master/setup.py#L183-L193) and [here](https://github.com/SciTools/iris/blob/master/setup.py#L198) 

I think this approach might be a step forward in the right direction, and perhaps prevent any future mishaps .. what do you think? :thinking: